nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/AbstractHiveQLProcessor.java

```diff
@@ -66,6 +71,38 @@
             .addValidator(StandardValidators.CHARACTER_SET_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor QUERY_TIMEOUT = new PropertyDescriptor.Builder()
+            .name("hive-query-timeout")
+            .displayName("Query timeout")
+            .description("Sets the number of seconds the driver will wait for a query to execute. "
+                    + "A value of 0 means no timeout. This feature is available starting with Hive 2.1.")
```

Comment:
I'd remove the 'This feature is available starting with Hive 2.1". Otherwise this all lgtm